pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "stactask"
-version = "0.5.1"
+version = "0.6.0"
 authors = [{ name = "Matthew Hanson", email = "matt.a.hanson@gmail.com" }]
 maintainers = [{ name = "Pete Gadomski", email = "pete.gadomski@gmail.com" }]
```

Comment:
@ircwaves can you remove me as a maintainer, and maybe add yourself or someone else? 🙇🏼

Reply:
To be clear, still happy to review stuff! I just don't feel like I'm in the position to be The Maintainer™ anymore :-P